runtime/ftplugin/modula2.vim

```diff
@@ -26,8 +26,12 @@ if exists("loaded_matchit") && !exists("b:match_words")
 endif
 
 if (has("gui_win32") || has("gui_gtk")) && !exists("b:browsefilter")
-  let b:browsefilter = "Modula-2 Source Files (*.def *.mod)\t*.def;*.mod\n" ..
-		     \ "All Files (*.*)\t*.*\n"
+  let b:browsefilter = "Modula-2 Source Files (*.def *.mod)\t*.def;*.mod\n"
```

Comment:
```suggestion
  let b:browsefilter = "Modula-2 Source Files (*.def, *.mod)\t*.def;*.mod\n"
```

I guess these were applied manually since this file diverged after 9.1.0013 :sob:

Reply:
Yes, exactly! Normally I would leave such files alone but that change was easy to make.

For the record, is the missing comma a problem? It wasn't there before, either.

Reply:
I don't think so, as that's just the text that's shown to the user (the patterns come after the tabs, which actually matter).

We can just remember to slip that comma in some time later :smile:

Reply:
(Well, in general, I don't think `b:browsefilter` matters in Nvim at all, except for keeping the files in sync)